<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>intensitymap</class>
 <widget class="QDialog" name="intensitymap">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>567</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>567</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>1000</width>
    <height>567</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Intensity Map</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="4">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Color Map</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="2">
      <widget class="QRadioButton" name="invertRB">
       <property name="text">
        <string>Invert</string>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0" rowspan="6">
      <widget class="QCustomPlot" name="intensityMap" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>700</width>
         <height>500</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="2">
      <widget class="QComboBox" name="colorPickDD">
       <item>
        <property name="text">
         <string>Polar</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Thermal</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Hot</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Cold</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Grayscale</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="4" column="1" colspan="4">
      <widget class="QTextBrowser" name="textBrowser"/>
     </item>
     <item row="5" column="4">
      <widget class="QPushButton" name="save_pb">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item row="3" column="4">
      <widget class="QRadioButton" name="interpolateRB">
       <property name="minimumSize">
        <size>
         <width>110</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Interpolate</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QSlider" name="rangeMaxSlider">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximum">
        <number>1024</number>
       </property>
       <property name="pageStep">
        <number>10</number>
       </property>
       <property name="value">
        <number>1024</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Cutoff</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>5</width>
         <height>10</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
